epoxy-adapter/src/main/java/com/airbnb/epoxy/BaseEpoxyAdapter.java

```diff
@@ -339,4 +348,12 @@ public boolean isStickyHeader(int position) {
   }
 
   //endregion
+
+  protected void executeWhenInflated(EpoxyViewHolder holder, Runnable runnable) {
+    if (holder.itemView instanceof AsyncInflatedView) {
```

Comment:
@varungu Have we considered making the async layout a property of the `EpoxyModel` or `EpoxyViewHolder` so we can avoid needing to do an `instanceof` check here?

Reply:
I have a TODO for this in the description. It is not very straight forward to make it a property. Biggest problem is all views will need to extend from this interface, that we don't want to do. If we don't enforce the extension, we will need the check in addition to the property.

My plan is to test this change before putting effort into adding a property.

Reply:
@varungu I just realized an additional concern with this approach. On each bind and attach/detach from the window this will do an object allocation of the runnable which may cause some performance hits for those not using async layouts. I think we'd probably want to better integrate this into the model or view holder before merging.

Reply:
We can separate this and create runnable only for AsyncViews, would that resolve your concern?

Reply:
Done.